src/js/swipe.js

```diff
@@ -0,0 +1,97 @@
+import { EVENTS } from './ui-events';
+import on from './on';
+
+class Swipe {
+  constructor(element) {
+    this.xDown = null;
+    this.yDown = null;
+    this.element = typeof (element) === 'string' ? document.querySelector(element) : element;
```

Comment:
We prefer to provide single argument type API's

Reply:
I will remove the 'string' option.